test/test_images/internal/knockdown/knockdown.go

```diff
@@ -42,7 +43,7 @@ type Config struct {
 // Main should be called by the process' main function. It will run the knockdown test. The return
 // value MUST be used in os.Exit().
 func Main(config Config, kdr Receiver) int {
-	client, err := cloudevents.NewDefaultClient()
+	client, err := kncloudevents.NewDefaultClient()
```

Comment:
I'm not sure that we should change the test image as we expect to always be given tracecontext headers.

Reply:
My reasoning is consistency. There is little downside to accepting both, so why not do it? The only real reason we wouldn't is to verify that the incoming headers truly are TraceContext, rather than B3. 

I also expect sooner or later someone will copy-paste this code and want the 'best practice' to be what they copy.

Reply:
Right my concern is detecting a regression where B3 is used instead of tracecontext. I also think that for normal event consumers accepting only tracecontext is good practice as that is what they should expect the broker to give them. On the other hand broker components should always use the clients package providers.

Reply:
Switched back with a comment explaining this is for tests only.